### ISSUE TITLE
Questions about desktop version

### ISSUE THREAD

**Issue by aukc1970** [Kooboo/Kooboo#5]
Good day !  Thanks for good job !  I would like to get more information about the desktop version.  The most important question is how to place, publish a website made on the desktop version on the hosting somee.com?
 What are the system requirements?  I would like to know more about Sync (push / pull) on a working example.  Thank you in advance for your response !

**Comment by koobooteam** [comment]
* First you need to host yourself.
We are providing Kooboo online free hosting, but still working in progress, but our monitoring system is not fully working now, it maybe more reliable to host yourself at this moment. 

regarding how to host yourself, please refer to another issue at #3 

* Sync tool 
There are many ways to publish your website live. Git alike Push/Pull is one of the option.  You can push your local website to online websites.  The doc can be found: 
https://www.kooboo.com/en/documentation/one_click_publishing

**Comment by koobooteam** [comment]
Looks like I can not answer here now? Github keep deleting my post????

**Comment by aukc1970** [comment]
What technologies does the development use new kooboo cms ? ASP.NET ? What programming language ?

**Comment by koobooteam** [comment]
It use .NET Framework 4.5.  Does not use ASP.NET, only the C# language. 

Any windows server (dedicated or cloud) can do. 

Linux version is under testing and not public available yet.

**Comment by koobooteam** [comment]
Not yet. 

All our online servers are running in the Desktop version too. 

Once it is available, we will first upgrade to our online servers. 

I do  not know we can make it in two weeks or not, that is what we planned. 

Regards,

Guoqi Zheng

**Comment by aukc1970** [comment]
Here's an interesting cross-platform solution :
https://ru.wikipedia.org/wiki/Mono
http://www.programbeginner.ru/?p=916

**Comment by koobooteam** [comment]
Your way of translation is correct.  This is considered as a bug.  We should not deleted your translation file during upgrade.  Can you create another issue with this bug so that our developer can track and fix it shortly. 

Linux version is for general Linux, we are testing on Ubuntu.  Testing is welcome and we should post here when we need it, we DO have 3000 unit test and a lot of automation UI testing, it is not much needed for now. 

You are free to offer development and hosting service based on Kooboo, or sell templates, kooboo applications.  we will have a market place where you can sell your service in very near future.

**Comment by koobooteam** [comment]
Yes, Mono is a good option. In the new .NET core, you can build a version for Linux directly as well. 

Kooboo linux verison is running on .NET Core.

**Comment by aukc1970** [comment]
I see ! Thank you very much for your answers ! Will the interface translation for Windows be the same for Linux ? or will I have to translate again ?The Linux version will be more up to date than windows because Linux servers are cheaper than windows and a Linux server free license like open source software means more accessible and legit.

**Comment by koobooteam** [comment]
Yes, everything working exactly the same on both Windows and Linux version.  Linux is  free, that is why we have to deliver a Linux version for community.

**Comment by aukc1970** [comment]
Thank You for Your work !

**Comment by koobooteam** [comment]
Hi, 

Do you think that you can try to commit your ru.xml files to the repository? So that we can include it in our next release? 
 
Under the folder of Kooboo.web/Lang/ru.xml.

The wrong deletion of translated language file has been fixed. Will be included in next release.

**Comment by aukc1970** [comment]
Good ! I haven't fully translated it yet. As translated to the end here can post a link to the translation file EN.xml. I translate with the help of this program http://quest-app.appspot.com/

**Comment by aukc1970** [comment]
I wanted to show about the translation into other languages in one project on github, can be useful !
https://github.com/turquoiseowl/i18n

**Comment by aukc1970** [comment]
It would be good if it was possible to generate language files using google translate. Then more languages could be covered.

**Comment by aukc1970** [comment]
I will translate until the end of the weekend on Saturday and Sunday.

**Comment by koobooteam** [comment]
great and thank you!

**Comment by aukc1970** [comment]
OK thanks !

**Comment by aukc1970** [comment]
Thanks for reply ! Sorry for the English ! Translate online translator. I wanted to say if you need help in testing Linux version that I could help. For which Linux version for desktop or server ? what packages does the Debian branch deb or rpm branch Red Hat have ? Interesting version for partners. What services should I offer ? Tried to translate the interface of the desktop version into Russian. Copied the en file.xml renamed to ru.xml in it translated but after updating the file disappeared was erased. How to translate correctly ? 22.10.2018, 09: 01, "Guoqi Zheng" <notifications@github.com>:Not yet.All our online servers are running in the Desktop version too.Once it is available, we will first upgrade to our online servers.I do not know we can make it in two weeks or not, that is what we planned.Regards,Guoqi Zheng—You are receiving this because you authorized the thread.Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by aukc1970** [comment]
Today I finished the translation EN.translated xml using this program http://quest-app.appspot.com/ and the google translate translation is certainly not perfect but then it will be possible to rule while working with the program. As promised here is the download link :
https://drive.google.com/file/d/1rIdgVfQkK-Jpa3cE5Tudc2FEEnaTAGGp/view?usp=sharing

**Comment by koobooteam** [comment]
Yes, we saw your ru.xml file in the repository now. thanks.
